scripts/console/2023.2.0/002_recover_vlans.py

```diff
@@ -7,6 +7,24 @@
 OF_LLDP_VLAN = 3799
 # Change to False to not print the missing VLANs
 PRINT_MISSING = True
+REMOVE_LLDP_FLOWS = False
+
+def get_cookie(dpid):
+    """Return the cookie integer given a dpid."""
+    COOKIE_PREFIX = 0xab
+    return (0x0000FFFFFFFFFFFF & int(dpid.replace(":", ""), 16) | (COOKIE_PREFIX << 56))
+
+def send_flows(dpid, cookie):
```

Comment:
Can you rename this as func remove_flows? To reflect more closely what it does.

Reply:
Renamed method and moved to `001_use_tags.py`